protocol/xmrmaker/claim.go

```diff
@@ -81,6 +88,40 @@ func (s *swapState) claimFunds() (*ethtypes.Receipt, error) {
 	return receipt, nil
 }
 
+// checkForMinClaimBalance check if we have enough balance to call claim.
+// return true if we do, false otherwise.
+func checkForMinClaimBalance(ctx context.Context, ec extethclient.EthClient) (bool, error) {
```

Comment:
I was about to suggest that this function belongs in the `relayer` package, but then I just looked at the `relayer` package to see where it would go, and there already is a function a named `checkForMinClaimBalance` in it.

Reply:
the one in `relayer` is specifically for claims that are relayed through the forwarder, it also has different return values and returns and error instead of true/false if there's not enough funds to relay. I don't think it makes sense to use that function here